scripts/localdev.py

```diff
@@ -502,8 +502,9 @@ def tempo_down(config):
               default='localdev',
               show_default=True,
               help="The test network's name")
-@click.option("--isolated-server-accounts",
-              is_flag=True,
+# Need to specify like this, or passing --isolated-server-accounts will cause isolated server accounts to be disabled
+# - rrw 2023-10-10
+@click.option("--isolated-server-accounts/--no-isolated-server-accounts",
```

Comment:
Not sure I quite understand this change.

Reply:
If you set default=True and then pass '--isolated-server-accounts', the effect is to turn isolated server accounts off. If you make the option explicit like this , that behaviour doesn't occur. I'll update the comment ..